gitops/snapshot_test.go

```diff
@@ -127,7 +127,7 @@ var _ = Describe("Gitops functions for managing Snapshots", Ordered, func() {
 	})
 
 	It("ensures the Snapshots status can be marked as invalid", func() {
-		gitops.SetSnapshotIntegrationStatusAsInvalid(hasSnapshot, "Test message")
+		gitops.SetSnapshotIntegrationStatusAsInvalidandNoRequeue(hasSnapshot, "Test message")
```

Comment:
Nit: capitalize "and" in function calls and function definition

Reply:
Thanks. Will change